lib/containers/k8s.pm

```diff
@@ -62,6 +62,7 @@ sub install_k3s {
     script_run("mkdir -p ~/.kube");
     script_run("rm -f ~/.kube/config");
     assert_script_run("ln -s /etc/rancher/k3s/k3s.yaml ~/.kube/config");
+    script_retry("kubectl get serviceaccount default -o name", delay => 20, retry => 10);
```

Comment:
I would increase the delay and decrease the retries, wdyt?

Reply:
If this is happening often, I would say that adding a `sleep 60` before the first try might be a good idea.

Reply:
added